Tests/Common/Service/UserAgentUtilTest.swift

```diff
@@ -39,4 +39,19 @@ class UserAgentUtilTest: UnitTest {
 
         XCTAssertEqual(expected, actual)
     }
+
+    func test_getUserAgent_givenDeviceInfoAvailable_expectLongVisonOsUserAgent() {
```

Comment:
I think this test is a duplicate of the `test_getUserAgent_givenDeviceInfoAvailable_expectLongUserAgent` test. Just with different mock values, which I don't think adds much value to the suite. 

Both test functions could be renamed to be more accurate of what logic is under test: *given device info values, expect the user agent is generated in the format that we expect*. 

If others think this is a valuable test, I am OK with it.

Reply:
I agree. My only justification for not changing it is that we only expect two variations. Keeping them explicit like this is easier to understand. If we introduce more, then it is much more justifiable to improve the test structure